TERRAFORM.md

```diff
@@ -157,3 +157,33 @@ make cfg=images/static/configs/alpine.apko.yaml apko-build-alpine
 ```
 
 These will build the image into a tarball in the local directory, which can be loaded into Docker with `docker load < image.tar`.
+
+## Creating new image scaffolding
+
+When creating a new image in this repository, you can run `monopod scaffold`
+from the root of this repository to generate a basic scaffolding:
+```shell
+monopod scaffold --package-name test-package --entrypoint /usr/bin/test-app
+```
+
+This will generate the following folder structure, providing you with a basic
+structure to get started building images:
+```console
+generated/
+└── test-package
+    ├── README.md
+    ├── configs
+    │   ├── latest.apko.yaml
+    │   └── main.tf
+    ├── image.yaml
+    ├── main.tf
+    └── tests
+        ├── EXAMPLE_TEST.sh
+        └── main.tf
+```
+
+You can then copy the whole `test-package` to the `images/` folder in this repository
```

Comment:
Not related to the docs for the thing, but wdyt about having `monopod scaffold` just write directly to `images/test-package` (and fail if it's already there)

Reply:
I can change the default to that. Currently it's possible but you have to specify the target folder as such.